components/organisms/CreateLessonWithinCourseAdminPortal/CreateLessonWithinCourseAdminPortal.tsx

```diff
@@ -184,8 +201,9 @@ export const CreateLessonWithinCourseAdminPortal = () => {
         <hr className={styles["button__split"]} />
         <div className={styles["button__content-delete-course"]}>
           <Button
-            className={styles["button__delete-course"]}
-            disabled={!selectedResource}
+            className={styles["button__submit"]}
+            // disabled={!selectedResource}
```

Comment:
do we need this commented line?

Reply:
That line is not necessary in the code, it was removed.